jerry-core/include/jerryscript-core.h

```diff
@@ -159,6 +159,18 @@ typedef struct
 } jerry_property_descriptor_t;
 
 /**
+ * Description of JerryScript heap memory stats.
+ * It is for memory profiling.
+ */
+typedef struct
+{
+  size_t version; /**< the version of the stats struct */
```

Comment:
But you don't know the changes in future versions. At compile time you see the correct version, but when dynamically linked, you get a library with different version.